src/main/java/org/broadinstitute/hellbender/utils/codecs/refseq/RefSeqCodec.java

```diff
@@ -0,0 +1,168 @@
+package org.broadinstitute.hellbender.utils.codecs.refseq;
+
+import htsjdk.tribble.AsciiFeatureCodec;
+import htsjdk.tribble.Feature;
+import htsjdk.tribble.TribbleException;
+import htsjdk.tribble.readers.LineIterator;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.Utils;
+
+import java.util.ArrayList;
+
+/**
+ * Allows for reading in RefSeq information
+ * TODO this header needs to be rewritten
+ * <p>
+ * Parses a sorted UCSC RefSeq file (see below) into relevant features: the gene name, the unique gene name (if multiple transcrips get separate entries), exons, gene start/stop, coding start/stop,
+ * strandedness of transcription. 
+ * </p>
+ *
+ * <p>
+ * Instructions for generating a RefSeq file for use with the RefSeq codec can be found on the documentation guide here
+ * <a href="https://gatkforums.broadinstitute.org/gatk/discussion/1329/where-can-i-get-a-gene-list-in-refseq-format">https://gatkforums.broadinstitute.org/gatk/discussion/1329/where-can-i-get-a-gene-list-in-refseq-format</a>
```

Comment:
url is broken. Should be : https://gatk.broadinstitute.org/hc/en-us/articles/360035532032-RefSeq-gene-list-format
Also user should be notified that file extension must be `.refseq`

Reply:
@oguzhankalyon Could you please file a new ticket about these issues instead of commenting on this merged pull request? Thanks!